packages/compass-workspaces/src/components/workspaces.tsx

```diff
@@ -82,8 +132,7 @@ type CompassWorkspacesProps = {
   onCreateTab(defaultTab?: OpenWorkspaceOptions | null): void;
   onCloseTab(at: number): void;
   onNamespaceNotFound(
-    tab: WorkspaceTab,
-    connectionId: string,
```

Comment:
Drive-by: no point in having connectionId explicitly in the interface where it's already part of the tab